boards/common/nrf52/include/periph_conf_common.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_nrf52
+ * @{
+ *
+ * @file
+ * @brief       Common peripheral configuration for nRF52 based boards
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ */
+
+#ifndef PERIPH_CONF_COMMON_H
+#define PERIPH_CONF_COMMON_H
+
+#include "periph_cpu.h"
+#include "cfg_clock_32_1.h"
```

Comment:
this seems wrong, it would mean that **all** nrf52-based boards use that clock config. This include should be placed in the board specific `periph_conf.h` files.